.github/workflows/renovate.yml

```
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Renovate
-        uses: renovatebot/github-action@v40.2.0
+        uses: renovatebot/github-action@v40.2.1
```

There's already new version https://github.com/renovatebot/github-action/releases/tag/v40.2.2
```suggestion
        uses: renovatebot/github-action@v40.2.2
```

---

thx